Lib/test/test_ctypes/test_free_threading.py

```diff
@@ -0,0 +1,89 @@
+from test.support import threading_helper
+import unittest
+import ctypes
+import threading
+
+threading_helper.requires_working_threading(module=True)
+
+
+class FreeThreadingTests(unittest.TestCase):
```

Comment:
My preference would be to get `test_ctypes` working with `--parallel-threads` instead of adding additional free-threading specific test cases.

You might need to mark some cases as `@support.thread_unsafe` if they use `warnings` or other non-thread safe code.